U/marina ricci/investigate properties of halos in dc2

marina-ricci opened the pull request:
This notebook is intended for the validation and visualisation of galaxy cluster properties in cosmoDC2, in the frame of HackUrDC2.
Clusters are defined as halos with M_fof>10^(14)Msol.

EiffL commented:
I took the liberty of fixing a coma in the notebook, which was preventing me from opening it (invalid json format).

rmandelb commented on contributed/check_properties_of_haloes_in_CosmoDC2.ipynb:
```diff
@@ -0,0 +1,659 @@
+   "metadata": {},
+   "source": [
+    "### Purpose of this notebook: \n",
+    "Objects in mock catalogs have to be representative of the their true population. In this context, this notebook aims at testing some basic statistical properties of galaxies belonging to clusters (arbitrarly defined here as halos with friend of friend mass > 10^14 Msol). The properties tested are : the mass/redshift distribution, the \"richness\"/mass relation, the density profile, the lumin ...
```
Minor typos: 
"the their" -> "their", "arbitrarly" -> "arbitrarily"

rmandelb commented on contributed/check_properties_of_haloes_in_CosmoDC2.ipynb:
```diff
@@ -0,0 +1,659 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load a calalog\n",
```
Minor typo: "calalog" -> "catalog"

rmandelb commented on contributed/check_properties_of_haloes_in_CosmoDC2.ipynb:
```diff
@@ -0,0 +1,659 @@
+   "outputs": [],
+   "source": [
+    "#define same cosmological parameters as the catalog\n",
+    "cosmo = FlatLambdaCDM(H0=71, Om0=0.2648, Ob0=0.0448) #not 100% sure it is the same cosmo as in the simu"
```
I wanted to point out that you can get the cosmological parameters to check whether these are correct as follows:
```
catalog1.cosmology
```
This will tell you the cosmological parameters, so you can check and then clarify this line.  ([This tutorial](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/extragalactic_gcr_hod.ipynb) has a nice usage of the catalog-based cosmological parameter access, which is used to create a CCL cosmology object, if you're interested to see how this works.)

rmandelb commented:
@marina-ricci -  we are looking to merge the HackurDC2-related PRs in the coming days. Might you be available to address the review comments by ~end of this week? Hopefully they can be done with quite minor modifications.

marina-ricci commented:
Thank you @rmandelb for your review! I can not do it at the moment but I plan to do it on friday.

marina-ricci commented:
I updated a newer version of the notebook and I took into account the comments.
I also corrected an error in the code and modified a bit the text. Thank you @rmandelb for the tip to load cosmology!

rmandelb commented:
@marina-ricci - thanks very much!  I've approved this and would like to go ahead and merge once you've cleared the notebook outputs.  Please let me know once that's ready...

marina-ricci commented:
I do not understand. Isn't it cleared already (there is not plot)?

rmandelb commented:
Sorry about that, you are correct.  Thanks, merging now!
